app/views/quizzes/_form.html.erb

```diff
@@ -9,11 +9,11 @@
 
 <%= form_with(model: @quiz, local: true) do |f| %>
   <p>
-    <%= f.label :title %><br>
+    <%= f.label t('title') %><br>
```

Comment:
use `activerecord` locales here.
google it, or take a look how it's implemented for books